src/main/java/com/icthh/xm/uaa/security/ldap/UaaLdapUserDetailsContextMapper.java

```diff
@@ -52,6 +55,7 @@ private void createUser(DirContextOperations ctx,
         //base info mapping
         userDTO.setFirstName(ctx.getStringAttribute(ldapConf.getAttribute().getFirstName()));
         userDTO.setLastName(ctx.getStringAttribute(ldapConf.getAttribute().getLastName()));
+        userDTO.setImageUrl(parseImageUrl(ctx));
```

Comment:
maybe make sense to add smth like UserMapperService with lep method, and add call lep method before createUser (line 63|67)
it will allow to add any map logic dynamic
(not only image, any additional information to filed data or etc)

Reply:
but it is only on user create. And on user update need either to edit java code or rewrite whole UaaLdapUserDetailsContextMapper, check each parameter(whether it changed or not) and update user